doc/reactions.md

```diff
@@ -8,166 +8,75 @@ Reactions allow SQL LRS to watch for patterns in submitted xAPI data and dynamic
-            "condition": "a",
-            "path": [
-              "timestamp"
-            ]
-          }
-        }
-      ]
-    }
-  },
-  "template": {
-    "actor": {
-      "mbox": {
-        "$templatePath": [
-          "a",
-          "actor",
-          "mbox"
-        ]
-      }
-    },
-    "verb": {
-      "id": "https://example.com/verbs/completed"
-    },
-    "object": {
-      "id": "https://example.com/activities/a-and-b",
-      "objectType": "Activity"
-    }
-  }
-}
+![reaction view 1](images/reactions/view_1.png)
+![reaction view 2](images/reactions/view_2.png)
+![reaction view 3](images/reactions/view_3.png)
+Each reaction has a title, ID, and created/modified timestamps. It also has an active vs. inactive status; a reaction can either be manually toggled as inactive, or it may be automatically set as inactive if an error is encountered when a reaction is triggered.
 
+The most important part of a reaction is its ruleset. Each ruleset in turn contains the following properties: conditions, template, and identity paths. Each ruleset component is explained in the sections below.
 
-```
+To create a new reaction, click on "Add New Reactions" on the main Reactions page. Likewise, to edit a current reaction, click on "Edit".
 
-### Identity Paths
-
-Identity Paths (`identityPaths` in the ruleset JSON) are a method of grouping statements for which you are attempting to match conditions. Typically, Reactions may revolve around actor Inverse Functional Identifiers (IFIs), e.g. `["actor", "mbox"]` or `["actor", "account", "name"]` which is equivalent to saying "For a given Actor, look for statements that share IFI values".
-
-Alternative approaches to Identity Path may be used by modifying `identityPaths`, for instance `["context", "registration"]` to group statements by learning session.
+![reaction edit](images/reactions/edit_intro.png)
+Creating a new reaction will display an edit page, where the user can modify the reaction title, active status, and ruleset.
 
 ### Conditions
 
-`conditions` is a mapping of names to rules for finding significant statements. Rules can be composed with boolean logic.
+Each condition is a set of rules for finding significant statements. Each condition has a unique name followed by its rules, which can be composed with boolean logic.
 
-In the example given above statement `a` must have an object id equal to `https://example.com/activities/a`, a verb id equal to `https://example.com/verbs/completed`, and a result success equal to `true`. Statement `b` must have the same verb and result success but an object id equal to `https://example.com/activities/b` and a timestamp greater than that of `a`.
+![reaction edit alpha](images/reactions/edit_condition_alpha.png)
+In the example given, statement "alpha" must have an object `id` equal to `https://example.com/activities/alpha`, a verb `id` equal to `https://example.com/verbs/completed`, and its result `success` property equal to `true`.
```

Comment:
just to illustrate the logic, I would write this as:

must have an object 'id' equal .... AND a verb 'id' equal to... AND 'result.success' property equal to

Reply:
https://github.com/yetanalytics/lrsql/pull/428/commits/6b0ffd2dba990ef3d2dca06324920f61f2102a8a